src/content/en/fundamentals/instant-and-offline/web-storage/index.md

```diff
@@ -179,15 +192,16 @@ many browsers as possible and which offer asynchronous call models, to maximize
 interoperability with the UI. These criteria lead naturally to the following
 technology choices:
 
-* For device local key/value storage use the Cache API.
+* For making your application itself available offline, use the Cache API. This API is available in
```

Comment:
This is a run-on sentence and hard to parse. Try something like:

For offline storage, use the Cache API. This API is available in any browser that supports Service Worker technology necessary for creating offline apps. The Cache API is ideal for storing resources associated with a known URL.